### DISCUSSION TITLE
exa/ls

### DISCUSSION THREAD

**Discussion by CallMeEchoCodes** [catppuccin/catppuccin#1865] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Maybe

**Additional context 📷 (optional)**
Basically just the `LS_COLORS` and `EXA_COLORS` environment variables

**Comment by nekowinston** [comment]
Would we use the terminal colours like `fg=red` or the hex values? exa/lsd are kinda themed by your terminal Emulator already, but I get what you mean. Extending it to more colours, probably, instead of the 10 terminal ones.

**Comment by BlueFalconHD** [comment]
https://the.exa.website/docs/colour-themes

**Comment by BlueFalconHD** [comment]
@CallMeEchoCodes pretty simple. Uses terminal color escape codes, so is theme-switch sensitive, and will work without catppuccin as well. Just need to decide what colors to use on different files. I have attached the default schemes. 
<img width="969" alt="image" src="https://user-images.githubusercontent.com/72631767/188330042-648a359d-4340-408b-99ba-512c0e98ac4d.png">

**Comment by raj-magesh** [comment]
Perhaps this project could be used: https://github.com/sharkdp/vivid

**Comment by yavko** [comment]
> Perhaps this project could be used: [sharkdp/vivid](https://github.com/sharkdp/vivid)

Ill make it rq

**Comment by yavko** [comment]
> > Perhaps this project could be used: [sharkdp/vivid](https://github.com/sharkdp/vivid)
> 
> Ill make it rq

I finished it but someone had already made a pr before me 💀

**Comment by yavko** [comment]
https://github.com/sharkdp/vivid/pull/94
